src/plugins/intel_cpu/src/transformations/cpu_opset/x64/op/llm_mlp.hpp

```diff
@@ -25,6 +25,7 @@ class LLMMLPNode : public ov::op::Op {
         int hidden_size;
         int up_size;
         bool gate_up_combined;
+        bool tail_f32 = false;
```

Comment:
maybe we can add `ov::element::Type output_type = ov::element::undefined;` instead, to be consistent with `FullyConnectedNode`

Reply:
> maybe we can add `ov::element::Type output_type = ov::element::undefined;` instead, to be consistent with `FullyConnectedNode`

@usstq I see. update the spec